tinkerbell/stack/templates/nginx.yaml

```diff
@@ -33,9 +33,6 @@ spec:
           envsubst '$POD_NAMESERVER' < /tmp/nginx.conf.template > /etc/nginx/nginx.conf;
           nginx -g 'daemon off;'
         ports:
-        {{- if not .Values.boots.hostNetwork }}
```

Comment:
Why is this now unnecessary?

Reply:
it's because nginx is no longer proxying for Boots